### PR TITLE
ci: change trigger for test automation from CircleCI to GitHub Actions

### PR DESCRIPTION
**Pull request by kaddaSz**
## Description
Changed the test automation trigger (cwa-app-automation) from circleCI to GitHub Actions.
- two new curl requests added in fastfile
- curl request to circleCI removed from fastfile

### CODE REVIEW

**Review by thomasaugsten — APPROVED** [review]

**Comment by kaddaSz** [comment]
@30mar or @puneetmahali1: could you please have a short look, why the UI tests are failing? I did only change something on the fastfile for adhoc distribution.

**Comment by puneetmahali1** [comment]
I also have the same problem yesterday, seems like some flaky UI Test. I have rerun the UI test failed job again.

**Comment by puneetmahali1** [comment]
@kaddaSz: It's passed and all checks looks green now.

**Review by puneetmahali1 — APPROVED** [review]

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/fastlane/Fastfile [hunk 1]
```diff
@@ -207,12 +207,25 @@ platform :ios do
...
+      -d '{\"ref\":\"main\", \"inputs\": { \"mobile-os\":\"ios\"}}' \
+      https://api.github.com/repos/corona-warn-app/cwa-app-automation/actions/workflows/31092010/dispatches"
+    )
+
+    sh("
+      curl \
+      -X POST \
+      -H 'Accept: application/vnd.github.v3+json' \
+      -H 'authorization: Bearer #{ENV["GITHUB_ACCESS_TOKEN"]}' \
+      -d '{\"ref\":\"main\", \"inputs\": { \"mobile-os\":\"ios125\"}}' \
```
Comment (naveeddotio) [inline src/xcode/fastlane/Fastfile:225]: What does iOS125 mean here? for version 12.5?

Comment (kaddaSz) [reply src/xcode/fastlane/Fastfile:225]: is an extra test automation for iOS 12.5 I guess. @thomasaugsten: can you comment on that?

Comment (thomasaugsten) [reply src/xcode/fastlane/Fastfile:225]: Yes correct it's a separate flow to run a complete test flow with iOS 12.5

**Pull request merged**